### PR TITLE
Switch to minikube instead of localkube for e2e tests

### PR DESCRIPTION
**Pull request by mbohlool**
WIP

### CODE REVIEW

**Comment by dims** [comment]
Looks like you are pretty close `1 failed` :)

**Comment by mbohlool** [comment]
Regardless of this works or not, I am worried about assert_hostname
failure. That means there is still a configuration that we get that error.

File: scripts/kube-init.sh [hunk 1]
```diff
@@ -39,64 +39,41 @@ else
...
+
+curl -Lo kubectl https://storage.googleapis.com/kubernetes-release/release/$(curl -s https://storage.googleapis.com/kubernetes-release/release/stable.txt)/bin/linux/amd64/kubectl && chmod +x kubectl && sudo mv kubectl /usr/local/bin/
+curl -Lo minikube https://storage.googleapis.com/minikube/releases/latest/minikube-linux-amd64 && chmod +x minikube
+
+export MINIKUBE_WANTUPDATENOTIFICATION=false
+export MINIKUBE_WANTREPORTERRORPROMPT=false
+export MINIKUBE_HOME=$HOME
+export CHANGE_MINIKUBE_NONE_USER=true
+mkdir $HOME/.kube || true
+touch $HOME/.kube/config
```
Comment (pokoli) [inline scripts/kube-init.sh]: Can we use a temp path for holding kube config? 

I remember losing all my .kube/config when running the e2e test on my local computer. 

As the config is exported latter, i do not see any drawback to use a non default hostpath.

Comment (mbohlool) [inline scripts/kube-init.sh]: This should not overwrite your existing config, but it is a good suggestion to be on the safe side.

**Pull request closed without merging**